packages/eslint-plugin/src/index.ts

```diff
@@ -8,18 +8,29 @@
 // Requirements
 //------------------------------------------------------------------------------
 
-const requireIndex = require('requireindex');
-const path = require('path');
+import requireIndex from 'requireindex';
+import path from 'path';
+
+import recommended from './configs/recommended.json';
 
 //------------------------------------------------------------------------------
 // Plugin Definition
 //------------------------------------------------------------------------------
 
+const rules = requireIndex(path.join(__dirname, 'rules'));
```

Comment:
we should consider dropping `requireIndex` and replacing it with statically generated list (it will be useful for making repl in web)

Reply:
but we should do that in separate PR.

Reply:
yeah I was considering doing it, but I couldn't be bothered with the double definition.
We can easily create a prebuild tool to scan and generate an `index.ts` in the rules folder.